### ISSUE TITLE
countloops function not defined 

### ISSUE THREAD

**Issue by ashutoshvt** [aasthan4/autogen#9]
I am running the follwing input script, 
```
from tests import ccsd_amplitude as ccsd
from main_tools.commutator import comm
from main_tools.product import prod
from library import print_terms
from library import full_con

#the input of the commutator function is as follows :
#if you want a commutator [V,T2]-> comm['op1'],['op2'],prefactor)
#NOTE : prefactor should be 0 in case this is not the last of the nested prefactor (clearer in the next eexample)

print 'case of [V,T1]'
list_terms=comm(['V2'],['T1'],1)
print_terms.print_terms(list_terms,'vt1test')

'''
print 'case of [V,T2]'
list_terms=comm(['V2'],['T2'],1)
#The following function selects only the fully contracted terms from the above list_terms
list_terms=full_con.full_terms(list_terms)
print_terms.print_terms(list_terms)

#Case of a nested commutator :
#Note that the name of the operators of the same type are different
#Like T1,T11,T12,T13 etc
print 'case of [[V,T1],T1]'
list_terms=comm(comm(['V2'],['T1'],0),['T11'],1.0)
```
but get the following error:
```
case of [V,T1]
there are contractions in this commutator
!!!!!!!!--------- ['p', 'q']
['r', 's']
['p', 'q', 'r', 's'] ['p', 'q'] ['r', 's']
['a', 'i'] ['a'] ['i']
doing contraction through multi_cont
0.25 fac is this for each term
0.25 fac is this for each term
Starting to compute sign of equivalent term2
into sign function
Traceback (most recent call last):
  File "vt1test.py", line 12, in <module>
    list_terms=comm(['V2'],['T1'],1)
  File "/Users/akumar1/autogen/main_tools/commutator.py", line 205, in comm
    cpre_env2.compare_envelope(list_terms, fc, last)    
  File "/Users/akumar1/autogen/library/compare_overall.py", line 13, in compare_envelope
    flo= cpre.compare(list_terms[i],list_terms[j])  
  File "/Users/akumar1/autogen/library/compare_test.py", line 78, in compare
    sign=level3_sign(term1,term2)
  File "/Users/akumar1/autogen/library/sign_function.py", line 16, in level3_sign
    loops1=countloops(term1)
NameError: global name 'countloops' is not defined
```
The function `countloops` seems to be not defined anywhere.

**Comment by asthanaa** [comment]
Can you please confirm that you are branch master?

**Comment by ashutoshvt** [comment]
Yes, I am on master

**Comment by asthanaa** [comment]
update

**Comment by ashutoshvt** [comment]
fixed through 618a2c66c4bf8c52dc96d213eabca10d4a1fbd31
